test.tsx

```diff
@@ -114,6 +114,18 @@ describe('react2angular', () => {
     it('should have empty bindings when parameter is not passed', () => {
       expect(react2angular(TestThree).bindings).toEqual({})
     })
+
+    it('should use the injectNames for DI', () => {
+      const defaultDi = (react2angular(TestThree).controller as any).slice(0, -1)
+      const injectedDi = (react2angular(TestThree, null, ['foo', 'bar']).controller as any).slice(0, -1)
+      expect(injectedDi).toEqual(defaultDi.concat(['foo', 'bar']))
```

Comment:
Instead of testing arity, please add broader tests. For example, you might consider injecting `$http`, spying on `$http.get`, and then calling it from the controller.

Also please add a test for a custom (not built-in) service: register it with `angular.service`, inject it, spy on it, and similarly make sure it gets called.

Reply:
Added tests using `$http`, `$q`, `$element` and a custom service. 
b357344
Also added tests for checking the priority of bindingNames, propTypes and injectedProps. 
be5c784